scripts/forger/seed/seed.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+# Select seed_dir
+seed_dir="${HOME}/.zen/seed"
+blocks_dir="${seed_dir}/blocks"
+chainstate_dir="${seed_dir}/chainstate"
+
+# Select data_dir
+data_dir="${HOME}/.zen"
+# Detect network
+if grep -q "testnet=1" "${HOME}/.zen/zen.conf" <(echo "${OPTS:-}"); then
+  data_dir+="/testnet3"
+fi
+
+# Main
+echo "Starting the seed process..."
+
+# Check if blocks_dir or chainstate_dir does not exist or is empty
+if [ ! -d "${blocks_dir}" ] || [ ! "$(ls -A "${blocks_dir}")" ] || [ ! -d "${chainstate_dir}" ] || [ ! "$(ls -A "${chainstate_dir}")" ]; then
+  echo "Either ${blocks_dir} or ${chainstate_dir} directories do not exist or are empty. Nothing to seed. Exiting script."
+  exit 0
+fi
```

Comment:
Isn't it that if directories are empty that means YES lets seed vs nothing to seed? :shrug:

Reply:
Not in this case, here the check is to find out if there is anything to seed, so these directories are the blocks and chainstate that come form the tar file and need to replace the blocks and chainstate of the node, so if they don't exist don't do anything

Reply:
Does it mean it checks if the tar file is kind of broken in this case?

Reply:
No, it checks if there is a seed to use. The logic is:
- when node starts it always runs the seed script
- If there is NOT a seed (blocks and chainstate directories exists in seed directory and are not empty) then it will exit the script because there is no seed.
- If there is a seed (blocks and chainstate directories exists in seed directory and are not empty) then:
- if .seed.complete file exists: exit the seed script
- If blocks or chainstate of the node's datadir exists exit script.
- If any of these exists and there is a seed and you want to force the re seed there is an env var to control it.

So the first time you run the zend node you may want to seed it, so you place the blocks and chainstate dirs in the seed dir.
But if you stop your nodes and start them again the seed folder will be empty because you don't want to reseed. 
This is what that first if is checking, is there a seed?

Reply:
By 'place the blocks and chainstate dirs in the seed dir` you mean like manually or by some other script/function if someone would populate those `/seed/{blocks,chainstate} dirs` and then run `seed.sh` ?

Reply:
Both. Process is described in the FORGER.md file. Now you can either use download_seed.sh file or do it manually